OasisDEX/oasis-borrow
# Modal Window - Update Your Signature

# [Modal Window - Update Your Signature](https://app.shortcut.com/oazo-apps/story/5664/modal-window-update-your-signature)

<please insert a shortcut link above>
  
## Changes 👷‍♀️

```
    if (!parsedToken.chainId) {
      return 'invalid'
    }
``` 
was added in jwt.ts - when `invalid` is returned `Update Your Signature` modal is shown and expired/invalid jwt is deleted from the local storage.
## How to test 🧪
you can check what is your `iat` in parsed  jwt and change `    if (!parsedToken.chainId) {` to `    if (parsedToken.iat === <your_current_iat>) {`

to simulate an invalid token that will get deleted - and the newly issued one will stay.
